Il2CppInspector.Common/Outputs/JSONMetadata.cs

```diff
@@ -261,6 +262,10 @@ public class JSONMetadata
             writer.WriteString("dotNetSignature", method.ToString().ToEscapedString());
         }
 
+        private void writeDeclaredTypeNamespace(MethodBase method) {
+            writer.WriteString("declared_type_namespace", method.DeclaringType.Namespace);
```

Comment:
Best to keep camelCase for the JSON keys for consistency: `declaringTypeNamespace`